### ISSUE TITLE
Option to not bundle Baritone

### ISSUE THREAD

**Issue by l1ving** [kami-blue/client#1764]
We should release optional `-nobaritone.jar` builds for things such as Future compatibility, which should still work if someone includes the Baritone API jar

**Comment by mvrozanti** [comment]
Are we able to manually replace the bundled version for another, for now?

**Comment by l1ving** [comment]
Yes. Obviously not including any baritone at all would break features.

Right now this is not as high priority as it was before because future has fixed it's issues with Baritone

**Comment by mvrozanti** [comment]
I mistyped my question;

Are we able to replace baritone's version post-compilation of kami-blue?

**Comment by l1ving** [comment]
Yes.

**Comment by mvrozanti** [comment]
How though? Unzip the post-compiled kami-blue.jar, unzip the baritone api jar and move the latter into the former? That doesnt seem ok, is there a better solution?

**Comment by l1ving** [comment]
No, just move the baritone API jar into your mods folder

**Comment by mvrozanti** [comment]
I'm gonna try that and report back

**Comment by mvrozanti** [comment]
Thanks that worked :)
